pyvisfile/vtk/vtk_ordering.py

```diff
@@ -50,25 +52,38 @@
 
 .. autofunction:: vtk_lagrange_quad_node_tuples
 .. autofunction:: vtk_lagrange_quad_node_tuples_to_permutation
-"""     # noqa
+"""     # noqa: E501
 
 
 # {{{
 
-def add_tuple_to_list(ary, x):
+@overload
+def add_tuple_to_list(ary: Sequence[Tuple[int, int]],
+                      x: Tuple[int, int]) -> Sequence[Tuple[int, int]]:
+    ...
+
+
+@overload
+def add_tuple_to_list(ary: Sequence[Tuple[int, int, int]],
+                      x: Tuple[int, int, int]) -> Sequence[Tuple[int, int, int]]:
+    ...
+
+
+def add_tuple_to_list(ary: Sequence[Tuple[int, ...]],
+                      x: Tuple[int, ...]) -> Sequence[Tuple[int, ...]]:
     return [add_tuples(x, y) for y in ary]
```

Comment:
I wasn't quite sure how to type this in a way that works. Maybe [TypeVarTuple](https://docs.python.org/3/library/typing.html#typing.TypeVarTuple) would do the trick? In the meantime, this works here..

Reply:
Totally fine with this, no need to overengineer.